Packs/CommonScripts/Scripts/IsUrlPartOfDomain/IsUrlPartOfDomain.py

```diff
@@ -0,0 +1,37 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+
+def main(domain_name: str, urls: str) -> CommandResults:
+    """Checks that the urls are in the domain in the domain name.
+
+    Args:
+        domain_name: A domain
+        urls: URLs to check if the domain in them.
+
+    Returns:
+        Results to display in CortexXSOAR
+    """
+    urls = argToList(urls)
+    results = demisto.executeCommand('ExtractDomainFromUrlAndEmail', {'input': domain_name})
```

Comment:
why are we even running that ? why not just run it for each url ?

Reply:
We're running it for each url.